src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -32,6 +32,7 @@ object JesBackend {
   lazy val GoogleApplicationName = JesConf.getString("applicationName")
```

Comment:
Perhaps out of scope here although the new changes are the ones which threw it over whatever mental threshold I had set up :)

Since we're really peeling a lot of stuff out of the config file for JES perhaps this would be another good use for validation, like how RuntimeAttributes does things?